File-Formats/PDF/Working-with-Compression.md

```diff
@@ -15,6 +15,9 @@ Essential PDF allows you to [compress the PDF](https://www.syncfusion.com/docume
 
 N> PDF supports compressing PDF document only in Windows Forms, WPF, ASP.NET and ASP.NET MVC platforms.
 
+To quickly get started with converting HTML to PDF in ASP.NET Core using the HTML-to-PDF Library. Please, check this video:
```

Comment:
For a swift introduction to converting HTML to PDF in ASP.NET Core with the HTML-to-PDF Library, take a look at this video tutorial: